.github/workflows/pr-tests.yml

```diff
@@ -203,12 +210,8 @@ jobs:
     - name: Run slow tests
       # Number threads is set to 1 because the runner does not have enough memory for more.
```

Comment:
Where does it actually limit the number of tests? We should try if it works with two threads.

Reply:
if i remember correctly, in nextest, enabling `no-capture` sets running threads to 1

Reply:
Yes, that's the answer.